warrior/Framework/Utils/data_Utils.py

```diff
@@ -1339,7 +1339,7 @@ def get_filepath_from_system(datafile, system_name, *args):
 
 def get_var_by_string_prefix(string):
     if "ENV" in string:
-        return os.environ[string.split('.')[1]]
+        return os.environ[string.split('.', 1)[1]]
     if "REPO" in string:
```

Comment:
change this condition and action to if string starts with 'REPO.' remove the first 5 chars and use the rest to fetch the value from data repository.